azure-iot-device/azure/iot/device/common/pipeline/pipeline_stages_base.py

```diff
@@ -500,28 +461,25 @@ def _do_connect(self, op):
         """
         # Alias to avoid overload within the callback below
         # CT-TODO: remove the need for this with better callback semantics
```

Comment:
Did you mean to leave this todo?

Reply:
Yeah this is fairly old. I can probably get rid of these, but I don't want to do it as part of this PR.